### ISSUE TITLE
Ensure PioneerRobot does not start before the sonars

### ISSUE THREAD

**Issue by aarant** [arantonitis/soar#9]
Occasionally, the ARCOS client will report that the sonars are enabled, while in reality it will take 1-2 seconds for the clicks to be heard. I don't think that this delay can be overcome, other than waiting on a sonar reading that isn't `None` before starting the brain, which has other undesirable side effects. To help mitigate the chances of this error occurring, a 1 second sleep will be added to the initial ARCOS connection, but I'm marking this as a wontfix.

**Comment by aarant** [comment]
Unless a better solution comes up, the delay introduced in 2495e8d should mitigate this issue.
